resources/views/community/components/active-players/active-players.blade.php

```diff
@@ -106,7 +106,7 @@ function activePlayersComponent() {
                 params.set('search', this.searchInput);
             }
 
-            if (getFullList || this.hasFetchedFullList) {
+            if (this.variant !== 'focused' && (getFullList || this.hasFetchedFullList)) {
```

Comment:
This doesn't do what it needs to. `all` just says to return 5000 rows instead of 20. You need to pass in the gameIds that were used in the initial query.

Reply:
Yep, you're right.

I think we should still keep this conditional, otherwise `?all=true` gets sent on the fetch URL. However, in latest, I've added some logic to send (and parse) the list of game IDs.